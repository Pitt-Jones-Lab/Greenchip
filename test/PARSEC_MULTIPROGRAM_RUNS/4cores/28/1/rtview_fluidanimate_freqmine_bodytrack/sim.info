{'benchmarks_revision': ('cat: .gitid: No such file or directory',),
 'benchmarks_root': ('/homes/grads/dek61/SNIPER/benchmarks',),
 'cmdline': ['/homes/grads/dek61/SNIPER/sniper/run-sniper',
             '-n',
             '4',
             '-m',
             'localhost',
             '-d',
             '/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/1/rtview_fluidanimate_freqmine_bodytrack',
             '-c',
             '/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_28_1_4.cfg',
             '-s',
             'stop-by-icount:10000000000',
             '--curdir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/1/rtview_fluidanimate_freqmine_bodytrack',
             '--trace-manual',
             '-g',
             '--traceinput/enabled=true',
             '-g',
             '--traceinput/emulate_syscalls=true',
             '-g',
             '--traceinput/num_apps=4',
             '-g',
             '--traceinput/trace_prefix=/tmp/tmpo1Uevu/run_benchmarks'],
 'git_revision': '68418dbe7cab15c65f1f4d300da0c505329beedd',
 'host': 'sb23',
 'pin_version': 'unknown',
 'rusage': (22612.859322,
            870.23870299999999,
            212188,
            0,
            0,
            0,
            49507,
            2,
            0,
            11608,
            1304,
            0,
            0,
            0,
            69194398,
            6289266),
 'snipercmd': ['bash',
               '-c',
               '/homes/grads/dek61/SNIPER/sniper/lib/sniper -c /homes/grads/dek61/SNIPER/sniper/config/base.cfg --general/total_cores=4 --general/output_dir=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/1/rtview_fluidanimate_freqmine_bodytrack --config=/homes/grads/dek61/SNIPER/sniper/config/nehalem28.cfg --config=/homes/grads/nbp3/SNIPER_CONFIGS/gainestown_28_1_4.cfg --traceinput/enabled=true --traceinput/emulate_syscalls=true --traceinput/num_apps=4 --traceinput/trace_prefix=/tmp/tmpo1Uevu/run_benchmarks -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -g --hooks/numscripts=1 -g --hooks/script0name=/scratch/nbp3/PARSEC_MUTLIPROGRAM_SHELL/4core/28/1/rtview_fluidanimate_freqmine_bodytrack/sim.scripts.py -g --hooks/script0args='],
 't_elapsed': 11211.091962814331,
 't_start': 1463065291.7106521,
 'tracecmds': [],
 'user': 'nbp3',
 'vmem': 1551462400L}
