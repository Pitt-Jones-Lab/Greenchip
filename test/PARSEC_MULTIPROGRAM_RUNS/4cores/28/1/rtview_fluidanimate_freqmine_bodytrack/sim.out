                                   | Core 0      | Core 1      | Core 2      | Core 3     
  Instructions                     |  2924158906 |  2695723907 |  2681442763 |  1697590389
  Cycles                           |  2459862398 |  2459862398 |  2459862398 |  2459862398
  IPC                              |        1.19 |        1.10 |        1.09 |        0.69
  Time (ns)                        |   924760300 |   924760300 |   924760300 |   924760300
  Idle time (ns)                   |      362935 |   188645816 |      341898 |   297850869
  Idle time (%)                    |        0.0% |       20.4% |        0.0% |       32.2%
Branch predictor stats             |             |             |             |            
  num correct                      |   352422413 |   316113803 |   375898309 |   168964951
  num incorrect                    |    30385512 |    20560534 |    26700894 |    20610468
  misprediction rate               |       7.94% |       6.11% |       6.63% |      10.87%
  mpki                             |       10.39 |        7.63 |        9.96 |       12.14
TLB Summary                        |             |             |             |            
  I-TLB                            |             |             |             |            
    num accesses                   |   405690027 |   348993971 |   342044669 |   193895536
    num misses                     |        5045 |         203 |         180 |         424
    miss rate                      |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                           |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                            |             |             |             |            
    num accesses                   |  1043082980 |   941636332 |  1127843889 |   515001447
    num misses                     |      483169 |     1173162 |      666013 |      533866
    miss rate                      |       0.05% |       0.12% |       0.06% |       0.10%
    mpki                           |        0.17 |        0.44 |        0.25 |        0.31
  L2 TLB                           |             |             |             |            
    num accesses                   |      488214 |     1173365 |      666193 |      534290
    num misses                     |      118364 |      327064 |      136846 |      112843
    miss rate                      |      24.24% |      27.87% |      20.54% |      21.12%
    mpki                           |        0.04 |        0.12 |        0.05 |        0.07
Cache Summary                      |             |             |             |            
  Cache L1-I                       |             |             |             |            
    num cache accesses             |   405690027 |   348993971 |   342044669 |   193895536
    num cache misses               |     5064910 |     1011569 |      966458 |     1002641
    miss rate                      |       1.25% |       0.29% |       0.28% |       0.52%
    mpki                           |        1.73 |        0.38 |        0.36 |        0.59
  Cache L1-D                       |             |             |             |            
    num cache accesses             |  1043082980 |   941636332 |  1127843889 |   515001447
    num cache misses               |     5121145 |     6532706 |     6694986 |     5331135
    miss rate                      |       0.49% |       0.69% |       0.59% |       1.04%
    mpki                           |        1.75 |        2.42 |        2.50 |        3.14
  Cache L2                         |             |             |             |            
    num cache accesses             |    10501423 |     7857542 |     8187100 |     6644347
    num cache misses               |     1855591 |     2291000 |     2042276 |     1501751
    miss rate                      |      17.67% |      29.16% |      24.95% |      22.60%
    mpki                           |        0.63 |        0.85 |        0.76 |        0.88
DRAM summary                       |             |             |             |            
  num dram accesses                |    11962164 |           0 |           0 |           0
  average dram access latency (ns) |      136.78 |         inf |         inf |         inf
  average dram queueing delay      |      136.78 |        0.00 |        0.00 |        0.00
Coherency Traffic                  |             |             |             |            
  num loads from dram              |     1243401 |     1202635 |     1470784 |     1049296
  num loads from dram cache        |           0 |           0 |           0 |           0
  num loads from remote cache      |           0 |           0 |           0 |           0
